.github/workflows/e2e-crossplatform.yml

```diff
@@ -0,0 +1,119 @@
+
+name: E2E cross platform
+
+on:
+  pull_request:
+
+jobs:
+  mac:
+    runs-on: macos-latest
+    timeout-minutes: 30
+    env:
+      TEST_SYSTEM: mac
+      ELECTRON_CUSTOM_VERSION: 19.0.5
```

Comment:
Is there a possibility to sync electron version with the one used in the release we're running tests against?

Reply:
there is a way to set environemnt variable dynamically - based on output of some command
like t his:

```
FOO="${{ github.event.inputs.foo }}"
echo "FOO=${FOO}" >> $GITHUB_ENV

```

Reply:
I will check it